14-draft.json

```diff
@@ -36,6 +36,10 @@
         "lon": {
           "description": "Longitude of your space location, in degree with decimal places. Use positive values for locations west of Greenwich, and negative values for locations east of Greenwich.",
           "type": "number"
+        },
+        "timezone": {
+          "description": "The timezone the space is located in. It should be formatted according to the <a target=\"_blank\" href=\"https://en.wikipedia.org/wiki/List_of_tz_database_time_zones\">TZ database location names</a>.",
```

Comment:
There is https://www.iana.org/time-zones , but they don't provide an HTML site, where all the names are listed.